.github/workflows/ui-tests.yml

```diff
@@ -22,18 +22,21 @@ concurrency:
 
 jobs:
   ui-tests:
-    name: UI Tests for ${{matrix.target}} on Simulators
+    name: UI Tests for ${{matrix.target}}
```

Comment:
`m`: Either leave any value of the matrix here, so they are not named identically when adding new matrix values, or remove the matrix entirely

Reply:
I'm confused. I have the `${{matrix.target}}` in the name. What should I change?

Reply:
My bad, it's fine like it is.